cfgov/core/services.py

```diff
@@ -1,51 +1,67 @@
-from django.views.generic.base import View
+import os
 
+from django.views.generic import View
+from django.http import HttpResponse
+from core.lib.PDFreactor import *
 
 
 class PDFGeneratorView(View):
     render_url = None
-    stylesheet_url=  None
+    stylesheet_url = None
     filename = None
+    license = os.environ.get('PDFREACTOR_LICENSE')
 
     def get_render_url(self):
         if self.render_url is None:
             raise ImproperlyConfigured(
                 "PDFGeneratorView requires either a definition of "
                 "'render_url' or an implementation of 'get_render_url()'")
+        return self.render_url
 
-    def get_stylesheer_url(self):
+    def get_stylesheet_url(self):
         if self.stylesheet_url is None:
             raise ImproperlyConfigured(
                 "PDFGeneratorView requires either a definition of "
-                "'stylesheer_url' or an implementation of 'get_stylesheer_url()'")
+                "'stylesheet_url' or an implementation of 'get_stylesheet_url()'")
+        return self.stylesheet_url
 
     def get_filename(self):
         if self.filename is None:
             raise ImproperlyConfigured(
                 "PDFGeneratorView requires either a definition of "
                 "'filename' or an implementation of 'get_filename()'")
+        return self.filename
+
+    def get_license(self):
```

Comment:
This seems like a weird application application of this pattern: I can't imagine a use case where we'd expect people to override the pdfreactor license in subclasses of this view.